src/Shared/Components/CICDHistory/TriggerOutput.tsx

```diff
@@ -404,45 +358,50 @@ const StartDetails = ({
             {triggerMetadata &&
                 renderDeploymentHistoryTriggerMetaText &&
                 renderDeploymentHistoryTriggerMetaText(triggerMetadata)}
+
             {isJobView && (
-                <div className="pt-4 pb-4 pr-0 pl-0">
-                    <span className="fw-6 fs-14">Env</span>
-                    <span className="fs-12 mb-4 ml-8">{environmentName !== '' ? environmentName : DEFAULT_ENV}</span>
-                    {environmentName === '' && <span className="fw-4 fs-11 ml-4 dc__italic-font-style">(Default)</span>}
+                <div className="flexbox dc__align-items-center dc__gap-4">
+                    <span className="cn-9 fs-13 fw-6 lh-20">Env</span>
+                    <span className="fs-12 lh-20">{environmentName !== '' ? environmentName : DEFAULT_ENV}</span>
+                    {environmentName === '' && <i className="fw-4 fs-11 lh-20">(Default)</i>}
                 </div>
             )}
         </div>
     )
 }
 
 const TriggerDetailsStatusIcon = React.memo(
-    ({ status, isDeploymentWindowInfo }: TriggerDetailsStatusIconType): JSX.Element => {
-        let viewBox = '0 0 25 87'
-        let height = '87'
-        let cyEndCircle = '74.5'
-        let y2Line = '69'
-        if (isDeploymentWindowInfo) {
-            viewBox = '0 0 25 118'
-            height = '118'
-            cyEndCircle = '105'
-            y2Line = '100'
-        }
-        return (
-            <svg width="25" height={height} viewBox={viewBox} fill="none" xmlns="http://www.w3.org/2000/svg">
-                <circle cx="12.5" cy="6.5" r="6" fill="white" stroke="#3B444C" />
-                <circle
-                    cx="12.5"
-                    cy={cyEndCircle}
-                    r="6"
-                    fill={colorMap[status]}
-                    stroke={colorMap[status]}
-                    strokeWidth="12"
-                    strokeOpacity="0.3"
-                />
-                <line x1="12.5" y1="11.9997" x2="12.5362" y2={y2Line} stroke="#3B444C" />
+    ({ status }: TriggerDetailsStatusIconType): JSX.Element => (
```

Comment:
Have separated the earlier svg into three parts containing two new svgs (self created svgs 💪), and a line through div and border, so that the height of component would be be derived by the container rather than fixed